packages/peregrine/lib/talons/CheckoutPage/PaymentInformation/useCreditCard.js

```diff
@@ -0,0 +1,439 @@
+import { useCallback, useEffect, useState, useMemo } from 'react';
+import { useFormState } from 'informed';
+import { useQuery, useApolloClient, useMutation } from '@apollo/react-hooks';
+
+import { useCartContext } from '../../../context/cart';
+
+export const mapAddressData = rawAddressData => {
+    if (rawAddressData) {
+        const {
+            firstName,
+            lastName,
+            city,
+            postalCode,
+            phoneNumber,
+            street,
+            country,
+            region
+        } = rawAddressData;
+
+        return {
+            firstName,
+            lastName,
+            city,
+            postalCode,
+            phoneNumber,
+            street1: street[0],
+            street2: street[1],
```

Comment:
I would consider maintaining these indices; labels are whatever, but the code should be consistent.

Reply:
I am sorry. I dint get this.

Reply:
Let's stick to `street1` and `street2`. Because `street0` and `street1` don't really make sense in the context of CSS classes, components, and labels. I wanted to maintain the same name codes actors to maintain consistency.